setup.cfg

```diff
@@ -4,13 +4,7 @@ max-line-length = 88
 extend-ignore = E203, W503
 
 [isort]
-include_trailing_comma = True
-known_third_party = django
-known_first_party = channels,daphne,asgiref
```

Comment:
Should we keep the 'known' settings to maintain the current ordering?

Reply:
I was wondering about that too. Let's decide later, after ☕️

Reply:
`daphne` and `asgiref` aren't really first party since they're not part of the package. Imo better to stick with defaults that are easy to keep consistent everywhere.

Reply:
OK, let's have it.